filters/ThirdParty/filter_253_IndianList/metadata.json

```diff
@@ -0,0 +1,16 @@
+{
+  "filterId": 253,
+  "name": "IndianList",
+  "description": "Additional filter list for websites in Indian.",
+  "timeAdded": 1650872701215,
+  "homepage": "https://github.com/mediumkreation/IndianList",
+  "expires": "4 days",
+  "displayNumber": 2,
+  "groupId": 7,
+  "subscriptionUrl": "https://easylist-downloads.adblockplus.org/indianlist.txt",
+  "tags": [
```

Comment:
Not recommended?

Reply:
Actually, this is the first time I've seen this filter, that's why I didn't add `recommended` tag.
On the other hand, I don't see any alternatives.